components/brave_rewards/resources/rewards_page/components/notification_modal.tsx

```diff
@@ -75,18 +75,41 @@ function Action(props: NotificationActionViewProps) {
   )
 }
 
+function useAutoDismissCheck(notification: Notification) {
+  const [externalWallet] = useAppState((state) => [state.externalWallet])
+
+  // Auto-dismiss "disconnected" notifications if the user is now connected.
+  if (notification.type === 'external-wallet-disconnected' &&
+      externalWallet && externalWallet.authenticated) {
+    return true;
```

Comment:
Total nit, but it seems like sometimes we're using a semicolon and sometimes not. Would be good to be consistent, I guess.

Reply:
Oops - muscle memory taking over